README.md

```diff
@@ -11,43 +11,189 @@ description: Creates an Azure AI Service and deploys the specified models.
 
-```bash
-# Log in to azd. Only required once per-install.
-azd auth login
+(Longer Description, as compared to the GitHub "about" section of the project)
+(make sure to highlight the use case!)
+Sample application code is included in this project. You can use or modify this app code or you can rip it out and include your own.
 
-# First-time project setup. Initialize a project in the current directory, using this template. 
-azd init --template Azure-Samples/azd-ai-starter
+[Features](#features) • [Gettting Started](#getting-started) • [Guidance](#guidance)
 
-# Provision and deploy to Azure
-azd up
-```
+(include a screenshot of your template's endpoint here-- so users know what it should look like when they're done)
 
-### Provisioned Azure Resources
+## Features
 
-This template creates the following resources:
+This project framework provides the following features:
 
-- [OpenAI Service](https://learn.microsoft.com/azure/ai-services/openai/)
+* Feature 1
+* Feature 2
+* ...
 
-The provisioning will also deploy any models specified within the `./infra/ai.yaml`.
+### Architecture Diagram
 
-For a list of supported models see [Azure OpenAI Service Models documentation](https://learn.microsoft.com/azure/ai-services/openai/concepts/models)
+Include a diagram describing the application. You can take [this image](https://raw.githubusercontent.com/Azure-Samples/serverless-chat-langchainjs/main/docs/images/architecture.drawio.png) as a reference.
 
-### Optional Configuration
+### Demo Video (optional)
 
-By default this template will use a default naming convention to prevent naming collisions within Azure.
-To override default naming conventions the following can be set.
+(Embed demo video here)
 
-- `AZURE_OPENAI_NAME` - The name of the Azure Open AI service
+## Getting Started
 
-Run `azd config set <key> <value>` after initializing the template to override the resource names
+You have a few options for getting started with this template. The quickest way to get started is [GitHub Codespaces](#github-codespaces), since it will setup all the tools for you, but you can also [set it up locally](#local-environment). You can also use a [VS Code dev container](#vs-code-dev-containers)
 
-## Reporting Issues and Feedback
+This template uses [MODEL 1] and [MODEL 2] which may not be available in all Azure regions. Check for [up-to-date region availability](https://learn.microsoft.com/azure/ai-services/openai/concepts/models#standard-deployment-model-availability) and select a region during deployment accordingly
 
-If you have any feature requests, issues, or areas for improvement, please [file an issue](https://aka.ms/azure-dev/issues). To keep up-to-date, ask questions, or share suggestions, join our [GitHub Discussions](https://aka.ms/azure-dev/discussions). You may also contact us via AzDevTeam@microsoft.com.
+  * We recommend using [SUGGESTED REGION]
+
+### GitHub Codespaces
```

Comment:
is github codespace required for every template?

Reply:
Yes, we want to promote the use of codespaces and devcontainers and the support for those development environments, over local setups, to reduce complexity. Getting it right for every OS, etc, is a lot more complex and error prone.